OWNERS_ALIASES

```diff
@@ -87,6 +92,30 @@ aliases:
   - spencerdillard
   - thisisnotapril
   - vaikas
+  knative-milestone-maintainers:
```

Comment:
I don't think we want this, and I'm curious why does it keep coming back? :)

Reply:
People can't run codegen, I fixed up this PR and submitted #845